modules/examples/eureka/manifests/fsx_static_provisioning.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: storage.k8s.io/v1

```

Comment:
you do not need a storage class for static provisioning. Here is an example:

https://github.com/kubernetes-sigs/aws-fsx-csi-driver/blob/master/examples/kubernetes/static_provisioning/README.md

Reply:
Okay. Sounds good. Updated

Reply:
Updated.